src/Pyz/Yves/CartPage/Theme/default/templates/page-layout-cart/page-layout-cart.twig

```diff
@@ -10,6 +10,8 @@
     request: app.request,
     title: 'cart.cart' | trans,
     requestUri: app.request.requestUri,
+    isCartItemsViaAjaxLoadEnabled: _view.isCartItemsViaAjaxLoadEnabled,
+    isUpsellingProductsViaAjaxEnabled: _view.isUpsellingProductsViaAjaxEnabled,
```

Comment:
Please extend the template from the core instead of copy-pasting

Reply:
I can't do it because the footer in the core is removed.

Reply:
IMO it's better to use default data from the core and reuse 'footer' block from 'page-layout-main'
@FRITS-Kh what do you think ?

Reply:
Looks like it makes sense because we have many equal parts at the core and can reuse them.